symfony/plugins/orangehrmAdminPlugin/Dao/CountryDao.php

```diff
@@ -69,11 +71,11 @@ public function searchCountries(array $searchParams) {
         try {
```

Comment:
Please convert whole class. `getProvinceList`, `searchCountries`

Reply:
I will do it when implementing the contact details screen

Reply:
Bcoz time allocated is not enough

Reply:
if you are not going to fix, add `// TODO` tag